socialdistribution/app/views.py

```diff
@@ -435,13 +436,22 @@ def create_comment(request, post_id):
     return render(request, 'comments/create_comment.html', {'form': form})
 
 @login_required
-def create_foreign_comment(request, post_id):
-    post = Node_Interface.get_post(data['post'])
-    token = Node.objects.get(url=post.author.url).auth_token
+def create_foreign_comment(request):
+    if request.method == 'POST':
+        return redirect('app:index')
```

Comment:
can we make this redirect back to the post, or the post comment page, instead of index?